main.py

```
@@ -9578,13 +9578,18 @@ async def lb_handler(interaction: discord.Interaction, type: str, do_edit: bool
         amount = 1
-    if cat_type not in cattypes:
+    if cat_type not in [*cattypes, "random"] or (cat_type == "random" and amount < 0):
         await message.response.send_message("bro what", ephemeral=True)
         return
 
     assert message.guild is not None
     user = await Profile.get_or_create(guild_id=message.guild.id, user_id=person_id.id)
-    user[f"cat_{cat_type}"] += amount
+    if cat_type == "random":
+        for _ in range(amount):
+            rolled_type = random.choices(cattypes, weights=list(data.type_dict.values()))[0]
```

Ruff (0.16.0)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)